diagnostic.md

```diff
@@ -27,7 +28,9 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+h1 {
+  font-size: $base-font-size: 30%;
```

Comment:
We want to increase the value of `$base-font-size` by 30% so it needs to be multiplied by 1.3.  The correct value is `font-size: $base-font-size * 1.3`.